dallinger/recruiters.py

```diff
@@ -727,6 +736,60 @@ def get_qualifications(self, hit_id, sandbox):
             "peripheral_requirements": details["peripheral_requirements"],
         }
 
+    def screen_out(self, participant: Participant):
+        """Screen-out a participant.
+
+        Parameters:
+            participant: Participant
+                The participant to screen-out.
+        """
+        if self.screen_out_allowed(participant):
```

Comment:
`screen_out_allowed` should only be used in DevProlificRecruiter. If we are in a real Prolific experiment, we should just make the call and see whether they reject it or not.

Reply:
It wasn't not clear to me why `screen_out_allowed` was only meant to be used in `DevProlificRecruiter`. See descrption here: https://gitlab.com/PsyNetDev/PsyNet/-/issues/881
Removing the `screen_out_allowed` condition for `ProlificRecruiter`

Reply:
OK, sorry it was not clear